assets/xml/objects/object_ob.xml

```diff
@@ -1,84 +1,84 @@
-        <Limb Name="object_ob_Standardlimb_016414" Type="Standard" EnumName="OBJECT_OB_LIMB_0A" Offset="0x16414" />
-        <Limb Name="object_ob_Standardlimb_016420" Type="Standard" EnumName="OBJECT_OB_LIMB_0B" Offset="0x16420" />
-        <Limb Name="object_ob_Standardlimb_01642C" Type="Standard" EnumName="OBJECT_OB_LIMB_0C" Offset="0x1642C" />
-        <Limb Name="object_ob_Standardlimb_016438" Type="Standard" EnumName="OBJECT_OB_LIMB_0D" Offset="0x16438" />
-        <Limb Name="object_ob_Standardlimb_016444" Type="Standard" EnumName="OBJECT_OB_LIMB_0E" Offset="0x16444" />
-        <Limb Name="object_ob_Standardlimb_016450" Type="Standard" EnumName="OBJECT_OB_LIMB_0F" Offset="0x16450" />
-        <Limb Name="object_ob_Standardlimb_01645C" Type="Standard" EnumName="OBJECT_OB_LIMB_10" Offset="0x1645C" />
-        <Limb Name="object_ob_Standardlimb_016468" Type="Standard" EnumName="OBJECT_OB_LIMB_11" Offset="0x16468" />
-        <Skeleton Name="object_ob_Skel_0164B8" Type="Flex" LimbType="Standard" LimbNone="OBJECT_OB_LIMB_NONE" LimbMax="OBJECT_OB_LIMB_MAX" EnumName="object_ob_Limbs" Offset="0x164B8" />
-        <Animation Name="object_ob_Anim_016F58" Offset="0x16F58" />
-        <Animation Name="object_ob_Anim_01764C" Offset="0x1764C" />
-        <Animation Name="object_ob_Anim_017E98" Offset="0x17E98" />
+        <DList Name="gMoonChildOdalwasMaskDL" Offset="0x690" />
+        <DList Name="gMoonChildGyorgsMaskDL" Offset="0x1D80" />
+        <DList Name="gMoonChildGohtsMaskDL" Offset="0x3AD0" />
+        <DList Name="gMoonChildTwinmoldsMaskDL" Offset="0x5020" />
+        <DList Name="gMoonChildMajorasMaskDL" Offset="0x6380" />
+        <DList Name="gMoonChildMajorasMaskEyesDL" Offset="0x6730" />
+        <Texture Name="gMoonChildMajorasMaskTex" OutName="moonchild_majoras_mask" Format="rgba16" Width="32" Height="64" Offset="0x67E0" />
+        <Texture Name="gMoonChildMajorasMaskSpike1Tex" OutName="moonchild_majoras_mask_spike_1" Format="rgba16" Width="16" Height="16" Offset="0x77E0" />
+        <Texture Name="gMoonChildMajorasMaskSpike2Tex" OutName="moonchild_majoras_mask_spike_2" Format="rgba16" Width="16" Height="16" Offset="0x79E0" />
+        <Texture Name="gMoonChildMajorasMaskSpike3Tex" OutName="moonchild_majoras_mask_spike_3" Format="rgba16" Width="16" Height="16" Offset="0x7BE0" />
+        <Texture Name="gMoonChildMajorasMaskSpike4Tex" OutName="moonchild_majoras_mask_spike_4" Format="rgba16" Width="16" Height="16" Offset="0x7DE0" />
+        <Texture Name="gMoonChildMajorasMaskInnerBorderTex" OutName="moonchild_majoras_mask_inner_border" Format="rgba16" Width="32" Height="32" Offset="0x7FE0" />
+        <Texture Name="gMoonChildMajorasMaskEyesTex" OutName="moonchild_majoras_mask_eyes" Format="rgba16" Width="32" Height="32" Offset="0x87E0" />
+        <Texture Name="gMoonChildOdalwasMaskTLUT1" OutName="moonchild_odalwas_mask_tlut_1" Format="rgba16" Width="4" Height="4" Offset="0x8FE0" />
```

Comment:
```suggestion
        <Texture Name="gMoonChildOdalwasMask1TLUT" OutName="moonchild_odalwas_mask_1_tlut" Format="rgba16" Width="4" Height="4" Offset="0x8FE0" />
```
Applies to the rest of the TLUTs as well, though if you can name the TLUT based on its usage that would be preferable (feel free to ping me in the discord if you would like help)

Reply:
why the TLUT word after the number? if you have 3 TLUTS for the same object, it reads as there are three objects:
gMoonChildOdalwasMask1TLUT
gMoonChildOdalwasMask2TLUT
gMoonChildOdalwasMask3TLUT
sounds like we have three identical masks, i vote for the number to be after the TLUT

Reply:
What's different between this and the textures where you've already moved the numbers before the suffix? That is all TLUT is, a suffix which is preferred to always be the suffix for every TLUT to easily tell what it is.

Reply:
Though naming them based on usage is still preferred over all, for example look at Odolowa's TLUT's. We see that `gMoonChildOdalwasMaskTLUT1` is always loaded just before `gMoonChildOdalwasMaskFaceTex` so could name it `gMoonChildOdalwasMaskFaceTLUT`. Again if you have questions or would like help naming the others just ping me in the discord.